docs/website/docs/general-usage/dashboard.md

```diff
@@ -31,9 +31,9 @@ You can even eject the code for the dashboard app into your current working dire
 
 ## Prerequisites
```

Comment:
I can't write a comment to `##Features` that is why I'm putting it here!

* Instead of `Features` let's use `Overview`

```
The Dashboard is a locally hosted application built on [Marimo](https://marimo.io/). It comes preconfigured with the key pipeline metadata and operational context you need to review pipeline health and validate data quality. 

With the Dashboard, you can:

1. Inspect pipeline metadata
2. Query data in the destination
3. Review traces and exceptions
4. Check the history of pipeline runs
5. Verify incremental loading behavior

You can also customize the Dashboard and create a personalized version tailored to your workflow.
```